### PR TITLE
feat: Add private/public repository, and improve help document 

### PR DESCRIPTION
**Pull request by liminxufen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,215 @@
...
+### Public Repository
+```hcl
+module "public_tcr" {
+  source          = "terraform-tencentcloud-modules/tcr/tencentcloud"
+
+  create_instnace = true
+  instance_type   = "Basic"
+  instance_name   = "your_instance_name"
+  open_public_operation = true
+  replications = {
```
Comment (Kagashino) [inline README.md]: 此处入参应为数组，如 
```
replications = [
  { regionId = 1, sync_tag = true}
]
```

下文和 `security_policies ` 同理

Comment (liminxufen) [reply README.md]: demo already updated

File: README.md [hunk 2]
```diff
@@ -4,6 +4,215 @@
...
+* [tencentcloud_tcr_vpc_attachment](https://registry.terraform.io/providers/tencentcloudstack/tencentcloud/latest/docs/resources/tcr_vpc_attachment)
+
+## Usage
+
+### Private Repository
+```hcl
+module "private_tcr" {
+  source          = "terraform-tencentcloud-modules/tcr/tencentcloud"
+
+  create_instnace = true
```
Comment (Kagashino) [inline README.md]: The variable name misspelled, it should be `create_instance`.

File: examples/README.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # Terraform TCR (Container Registry) example
 
-This sample demostrates how to create a TCR instance, attach it into a VPC/Subnet, and create multiple namespaces and multiple repositories. The user_name and token are also exported in the output.
+This sample demostrates how to create a TCR instance, attach it into a VPC/Subnet, and create namespace and repository. The instance_name and repository url are also exported in the output.
```
Comment (Kagashino) [inline examples/README.md]: Another misspelled, it should be `demonstrates `.

**Review by Kagashino — APPROVED** [review]

**Pull request merged**